README.md

```diff
@@ -1,3 +1,9 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+/\*
+In order to run this file locally, create the following 2 files: .env.test AND .env.development, with a single line of code that references the appropriate database.
+
+eg.
+
+.env.development: PGDATABASE=nc_news
+.env.test: PGDATABASE=nc_news_test
```

Comment:
Good explicit instructions, might be good to make it even clearer by saying that the line of code is only `PGDATABASE=nc_news` (etc.) and doesn't include `.env.development:`